test/primitives-test.js

```diff
@@ -17,10 +17,10 @@ describe('Minim Primitives', function() {
       });
 
       it('should initialize the correct meta data', function() {
-        expect(el.meta.id.toValue()).to.equal('foobar');
```

Comment:
Shouldn't we still keep the verbose variant of the test to expose the actual API since the helper methods are just that. Or are we no longer publicly supporting this API?